server/plugins/client/bundle-handler-factory.js

```diff
@@ -0,0 +1,54 @@
+module.exports.createHandler = createHandler
```

Comment:
why not simply `module.exports = createHandler`? That’s what we usually do: 1 file - 1 method :)